test/go.mod

```diff
@@ -4,6 +4,8 @@ go 1.22.0
 
 replace sigs.k8s.io/cluster-api => ../
 
+replace sigs.k8s.io/controller-runtime => github.com/sbueringer/controller-runtime v0.2.0-beta.1.0.20240528125616-c6053f022f6b
```

Comment:
This version of CR prints a stacktrace whenever an unstructured uncached get/list is run